tests/test_mfa.py

```diff
@@ -0,0 +1,198 @@
+from workos.mfa import MFA
+import pytest
+
+
+class TestWebhooks(object):
```

Comment:
Looks like this name was copied over:

```suggestion
class TestMfa(object):
```

Reply:
✔️